package.json

```diff
@@ -6,9 +6,13 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.9.0",
+    "react-router-dom": "^6.11.2",
     "react-scripts": "5.0.1",
+    "sass": "^1.62.1",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] Great job so far everything looks fine 👍🏻 , however, I'm getting this error after installing the project dependencies 👇🏻 Please fix this. 👍🏻 

![image](https://github.com/BenM4k/react-bookstore/assets/79058364/8e94f759-fd11-4fff-91c5-9f53472e56ff)